src/components/UI/Dropdown/index.js

```diff
@@ -33,8 +33,15 @@ export function StyledDropdown(props) {
   return (
     <DropDownContainer>
       <DropDownHeader onClick={handleOpen}>
-        {selectedOption?.name}
+        {selectedOption?.icon ? (
+          <Icon
+            src={selectedOption.icon ? selectedOption.icon : undefined}
```

Comment:
No need for a ternary operator here, `selectedOption?.icon` evaluates either to an icon or to `undefined`, so for `src` it is safe to use `selectedOption.icon` directly